content/en/docs/components/pipelines/sdk/build-pipeline.md

```diff
@@ -415,7 +398,9 @@ def my_pipeline(url):
 
 ### Compile and run your pipeline
```

Comment:
Please review this following general section to introduce how to run a pipeline.
When this is reviewed, I'll copy this part to all the docs with a similar section.